### PR TITLE
Feature/geode 2014

### PR DESCRIPTION
**Pull request by kjduling**
Updates to the LICENSE file

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -329,8 +302,6 @@ Apache Geode bundles the following files under the MIT license:
     Copyright (c) 2011 Sencha Inc.
   - jQuery JavaScript Library v1.7.2, v1.8.0 (https://jquery.com), Copyright (c)
```
Comment (jinmeiliao) [inline LICENSE]: I believe you removed jquery1.8.0. jQuery 1.7.2 is used by pulse.

File: geode-assembly/src/test/resources/expected_jars.txt [hunk 2]
```diff
@@ -37,10 +37,6 @@ jline
 jna
```
Comment (jinmeiliao) [inline geode-assembly/src/test/resources/expected_jars.txt]: Why is this file change in this pull request? This change is already handled by your last pull request.

File: LICENSE (hunk 1 above)
Comment (kjduling) [inline LICENSE]: That's correct.  1.8 was in the swagger-ui and no longer used.

Comment (kjduling) [inline LICENSE]: Oh I see.  Yeah.  1.8 and 1.8 for UI had to be removed.  I only got the 1.8 UI

**Pull request closed without merging**